Lib/fontTools/otlLib/builder.py

```diff
@@ -244,11 +251,70 @@ def add_subtable_break(self, location):
         self.alternates[(self.SUBTABLE_BREAK_, location)] = self.SUBTABLE_BREAK_
 
 
+class ChainContextualRuleset():
+    def __init__(self):
+        self.rules = []
+
+    def addRule(self, prefix, glyphs, suffix, lookups):
+        self.rules.append((prefix, glyphs, suffix, lookups))
+
+    @property
+    def hasPrefixOrSuffix(self):
+        # Do we have any prefixes/suffixes? If this is False for all
+        # rulesets, we can express the whole lookup as GPOS5/GSUB7.
+        for (prefix, glyphs, suffix, lookups) in self.rules:
+            if len(prefix) > 0 or len(suffix) > 0: return True
+        return False
+
+    @property
+    def hasAnyGlyphClasses(self):
+        # Do we use glyph classes anywhere in the rules? If this is False
+        # we can express this subtable as a Format 1.
+        for (prefix, glyphs, suffix, lookups) in self.rules:
+            for coverage in (prefix, glyphs, suffix):
+                if any([len(x) > 1 for x in coverage]):
+                    return True
+        return False
+
+    def format2Classdefs(self):
+        PREFIX, GLYPHS, SUFFIX = 0,1,2
+        classdefbuilders = []
```

Comment:
```suggestion
        classDefBuilders = []
```

I think we should stick to one style for the entire module. If the rest uses camelCase, then new code should follow.